### DISCUSSION TITLE
installing landingzone

### DISCUSSION THREAD

**Discussion by yillouz-fortinet** [GoogleCloudPlatform/pbmm-on-gcp-onboarding#248] [Q&A]
hi,
I'm new on this and trying to install the product for the first time .
can you direct me to some documentation explaining / show example of the fields needed in common.auto.tfvars
i.e. forlder_iam.member
audit_project_iam.member

whats the meaning of these fields and expected values ?

**Comment by fmichaelobrien** [comment]
Hi, 
- see last full run 20220521 (pre TF 1.3.7 upgrade) of the LZ with config examples https://github.com/GoogleCloudPlatform/pbmm-on-gcp-onboarding/issues/97
- and a later partial update in 20221122 https://github.com/GoogleCloudPlatform/pbmm-on-gcp-onboarding/pull/208
- 

For the config - see 97 - you can put the same super admin account across all user: and group: emails in a dev env.
```
folder_iam = [
   {
-    member = "group:group@test.domain.net" # REQUIRED EDIT. user:user@google.com, group:users@google.com,serviceAccount:robot@PROJECT.iam.gserviceaccount.com
+    member = "user:root@clo..ge.dev" # REQUIRED EDIT. user:user@google.com, group:users@google.com,serviceAccount:robot@PROJECT.iam.gserviceaccount.com


-audit_project_iam = [ #REQUIRED EDIT. At leave one object is required. The member cannot be the same for multiple objects.
+audit_project_iam = [ #REQUIRED EDIT. At least one object is required. The member cannot be the same for multiple objects.
   {
-    member = "user:group@test.domain.net" #REQUIRED EDIT
+    member = "user:root@cl...ge.dev" #REQUIRED EDIT
```

However
I recommend switching over to the LZ V2 written around the Kubernetes based Config Controller (wraps Terraform) being actively worked on in the following repo

https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/tree/main/docs/landing-zone-v2 
specifically the perimeter work in https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/issues/258
